packages/expo-modules-core/android/src/main/java/expo/modules/adapters/react/services/UIManagerModuleWrapper.java

```diff
@@ -98,6 +101,16 @@ public View get(Object key) {
     });
   }
 
+  @Nullable
+  @Override
+  public View resolveView(int viewTag) {
+    final com.facebook.react.bridge.UIManager uiManager = UIManagerHelper.getUIManagerForReactTag(getContext(), viewTag);
```

Comment:
Can't we import it?

Reply:
> Can't we import it?

there is already a UIManager import
https://github.com/expo/expo/pull/18823/files/84c18625e00b2fe6865c86cc9ef7ad61bf104920#diff-5afbbe6152e96fc1b09937cf6c760454be89ea5d9743abf50e61a6034292f908R28